### PR TITLE
Refactor Folder Stucture

### PR DESCRIPTION
**Pull request by yee2542**
break changes on:
- `__test__` -> tests
- `*` -> /src
- form prev `stores` will keep all types of reducers now will be move to each `Modules` e.g. `common/stores/userReducer -> modules/user/reducers`
- `login` page will be parts of `user`
- move `ant css` from `_app.tsx -> Styles/global.styled.ts`
- hard coded model with constant will move to `*/constants` e.g. `Models/menu -> Modules/home/constants`

add new
- `src/common`
- `src/modules`
- `src/pages` **for `next/pages`
- `src/common/pages` ** for `react` components
- alias `Modules` point to `src/modules`

for user module will move here
- `src/modules/user`

deprecated
- ~~on `ci.yml` decide to remove `test:ci` command~~ move back to `test:ci`

### CODE REVIEW

**Review by octobrekids — COMMENTED** [review]
Fix then gogo!

File: stories/LoginPage.stories.tsx [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
   title: 'FluentSearch/LoginPage',
-  component: Login,
+  component: LoginPage,
 } as Meta;
 
-const Template: Story = (args) => <Login {...args} />;
+const Template: Story = (args) => <LoginPage {...args} />;
 
-export const LoginPage = Template.bind({});
+export const LoginStories = Template.bind({});
```
[the remark this answers is not in the document]
Comment (yee2542) [reply stories/LoginPage.stories.tsx:11]: yep, fix it

File: src/modules/user/pages/login/types.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { FormProps } from 'antd/lib/form/Form';
+
+export type FormLogin = { email: string; password: string };
+export type FormLoginError = { values: FormLogin; errorFields: any[] };
```
[the remark this answers is not in the document]
Comment (yee2542) [reply src/modules/user/pages/login/types.ts]: change to `unknown` and add todo to remark it

File: stories/LoginPage.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (octobrekids) [reply stories/LoginPage.stories.tsx]: Fix here  aaac4774e1dd6191e3d61d0f5a8829d08aee17a3

**Review by octobrekids — APPROVED** [review]
Gogo!

**Pull request merged**